tests/specs/node/byonm_phantom_dep_res_failure/bad_import.out

```diff
@@ -0,0 +1,2 @@
+error: [ERR_MODULE_NOT_FOUND] Cannot find module 'file:///[WILDLINE]/node_modules/package/index.js' imported from 'file:///[WILDLINE]/bad_import.ts'
+    at file:///[WILDLINE]/bad_import.ts:1:16
```

Comment:
We now get proper node resolution errors here. This demonstrates a simple case, but imagine if this was something like exports not being resolved or the like. One issue is that the `imported from '...'` is repetitive because deno graph provides this information. We may want to consider removing the referrer from error messages and then only adding it when necesasry (ex. when importing esm with `require`)

Before:

```
error: Cannot find "package"
    at file:///V:/deno/tests/specs/node/byonm_phantom_dep_res_failure/bad_import.ts:1:16

```

Reply:
I think it's fine for this pass 👍